static/templates/settings/upgrade_tip_widget.hbs

```diff
@@ -1,7 +1,14 @@
 <div>
     {{#unless zulip_plan_is_not_limited}}
     <a href="/upgrade" class="upgrade-tip" target="_blank" rel="noopener noreferrer">
+    {{#if is_business_type_org}}
         {{upgrade_text_for_wide_organization_logo}}
+    {{else}}
+		{{#tr}}
+		Available on Zulip Cloud Standard. Upgrade to access or <z-link-sponsorship>request sponsorship.</z-link-sponsorship>
+		{{#*inline "z-link-sponsorship"}}<object><a href="/upgrade/#sponsorship">{{> @partial-block}}</a></object>{{/inline}}
```

Comment:
I have used `<object>` here, since we can't nest <a> elements. But object is intended to use for multimedia elements like image, video. There can be another solution using some CSS workarounds or avoid using `<a>` tag and instead use JS to make it a clickable link, but not sure what we would prefer.

Reply:
@andersk can you help with this question? -- I'd prefer to avoid using object tags, but am not sure what the best alternative solution is.

Reply:
Why again are we trying to nest links inside of each other, instead of…not doing that? Move the outer `<a>` into the `{{if}}` branch, and write the `{{else}}` branch using two non-nested links: “Available on Zulip Cloud Standard. [Upgrade to access] or [request sponsorship].”

Reply:
We can do this and that would mean the whole bannner will not be linked for the `else` case. Sounds reasonable. @alya thoughts?

Reply:
Yep, works for me. That's probably more straightforward from a UX perspective in any case.